### PR TITLE
Updated documentation for OpenWeatherMap alongside PR 39839

### PR DESCRIPTION
**Pull request by nzapponi**
## Proposed change

See PR https://github.com/home-assistant/core/pull/39839

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/39839
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/openweathermap.markdown [hunk 1]
```diff
@@ -42,10 +42,10 @@ name:
   type: string
   default: OpenWeatherMap
 mode:
-  description: "Can specify `hourly`, `daily`, or `freedaily`. Select `hourly` for a three-hour forecast, `daily` for daily forecast, or `freedaily` for a five-day forecast with the free tier."
+  description: "Can specify `hourly` or `daily`."
```
Comment (frenck) [inline source/_integrations/openweathermap.markdown]: We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>
- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Furthermore, there is a merge conflict and there are unrelated commits in this PR. Could you please clean it up?

File: source/_integrations/openweathermap.markdown (hunk 1 above)
Comment (nzapponi) [reply source/_integrations/openweathermap.markdown]: Resolved

**Comment by nzapponi** [comment]
@frenck addressed the requested change by resolving the conflict that was there already

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Comment by nzapponi** [comment]
Hi @frenck, actually this is still relevant, see PR https://github.com/home-assistant/core/pull/39839 .
I have updated the main request at the top.

Can you please reopen? Or should I create a new one?

**Review by frenck — APPROVED** [review]
Thanks, @nzapponi 👍

**Pull request merged**